extensions/wikia/AdEngine/js/provider/openX.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Experimental ad provider for OpenX. It loads the OpenX code in a (non-GPT) iframe.
+ * To preview how it would work, use ?forceopenx=1 URL param
+ *
+ * Corresponding DFP Lines (ztest OpenX CC ** Entertainment INTL):
+ *
+ * INCONTENT_BOXAD_1: https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112107492
+ * TOP_LEADERBOARD:   https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112107612
+ * TOP_RIGHT_BOXAD:   https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112107132
+ * PREFOOTER:         https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112107252
+ * LEFT_SKYSCRAPER:   https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112107372
+ * MOBILE_TOP_LEADER: https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112963692
+ * MOBILE_PREFOOTER:  https://www.google.com/dfp/5441#delivery/LineItemDetail/orderId=268596372&lineItemId=112963932
+ */
+
+/*global define*/
+define('ext.wikia.adEngine.provider.openX', [
+	'wikia.document',
+	'wikia.iframeWriter',
+	'wikia.log'
+], function (doc, iframeWriter, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.provider.openX',
+		slotMap = {
+			'HOME_TOP_LEADERBOARD':   {size: '728x90',  auid: 537201006},
```

Comment:
Oh, OK. I've got it. I can see it being used and how its being used below. Thanks! ;)